AraEvent/AraEventCalibrator.cxx

```diff
@@ -871,18 +901,89 @@ void AraEventCalibrator::calibrateEvent(UsefulAtriStationEvent *theEvent, AraCal
         loadAtriPedestals(thisStationId);
     }
 
-    //Step one is loop over the blocks
+    // 3rd step. Converts DAQ data format to Electronic channel format
+    // Bin trimming and Pedestal subtraction are now in the seperate functions
+    DAQtoEleCh(theEvent, voltMapIt, timeMapIt, sampleList, capArrayList);
+
+    // From here, each function is independent. So, You can call the functiions at any step
+    // As a default, prioritizing the functions that reduce the number of samples
+    
+    // 4th step. Erase first block that currupted by trigger
+    // Apply conditioner function here for the correct ZeroMean() and VoltCal() performance
+    if(hasTrimFirstBlock(calType)) {
+        hasTrimFirstBlk = TrimFirstBlock(theEvent, voltMapIt, timeMapIt, sampleList, capArrayList);
+    }    
+
+    // 5th step. Trims samples using fAtriSampleIndex table
+    // A trimmed sampleList will be used for calling the right pedestal subtraction samples and the conversion factor samples
```

Comment:
Add a little more description here. What does BinWidthCalib mean? What does "trimmed sample list" mean?

Reply:
This part is essentially giving the correct dT (time width between samples) to each capacitor sample based on the values in the table. So, maybe `Timing calibration` would be a more intuitive name. 
During the process, it removes (trimming out) the index of badly performing capacitor samples from the `sampleList` which is the total indexes of capacitor samples used for recoding the data.
And it also removes ADCs that are recorded in badly performing capacitor samples.
In the later step, such as voltage calibration, the correct index to the corresponding ADC value is need to call the right conversion factor.